### ISSUE TITLE
layouts.all.all instead of layouts.all?

### ISSUE THREAD

**Issue by dpieri** [mapbox/maki#455]
I'm using version 6.1.0 within an app written in Coffeescript.

The documentation indicates that you can iterate through `maki.layouts.all` but in my experience `maki.layouts.all` is an object with a key `all`, so to iterate through the icons you need to access `maki.layouts.all.all`.

Is this something specific to my coffeescript setup, or does the documentation need to be updated?

```coffeescript
maki = require '@mapbox/maki'

maki.layouts.all # { all: [...] }
maki.layouts.all.all # [...]
```

**Comment by tristen** [comment]
Thanks for the issue! This looks to be an issue and something that should be solved from the exported reference over a doc update. Submitted a PR in #457 to address this 👍

**Comment by dpieri** [comment]
Is this going to bump the major version from 6 to 7?

**Comment by tristen** [comment]
> Is this going to bump the major version from 6 to 7?

@dpieri that's correct.

**Comment by tristen** [comment]
Will be closed in the next release by #457
